lib/src/main/java/com/diffplug/spotless/sql/CommonUtils.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.sql;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Common utils
+ */
+public class CommonUtils {
+
+	public static boolean isEmpty(CharSequence value) {
+		return value == null || value.length() == 0;
+	}
+
+	public static boolean isEmpty(String value) {
+		return value == null || value.length() == 0;
+	}
+
+	public static boolean isEmpty(Collection<?> value) {
+		return value == null || value.isEmpty();
+	}
+
+	public static boolean isEmpty(Map<?, ?> value) {
+		return value == null || value.isEmpty();
+	}
+
+	public static boolean equalObjects(Object o1, Object o2) {
```

Comment:
`Objects.equals` would be fine but I think that was introduced in Java `1.7`.

Reply:
As said in the description. the code is really messy, I will clean it. I first need some help on the few points I mentioned above.